edge-util/src/Microsoft.Azure.Devices.Edge.Storage.RocksDb/disk/DiskSpaceChecker.cs

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Storage.RocksDb.Disk
+{
+    using System;
+    using System.IO;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Storage.Disk;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+
+    public class DiskSpaceChecker : IStorageSpaceChecker
+    {
+        static readonly int checkFrequencyDefault = 120;
```

Comment:
>static readonly [](start = 8, length = 15)

Can be a const instead and rename to begin with an uppercase C if you are changing it